### ISSUE TITLE
Node gets cleared out when clicking on the blank canvas space while editing this node

### ISSUE THREAD

**Issue by xvcgreg** [enso-org/enso#6545]
### What type of issue is this?

Intermittent – Occurring irregularly

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [X] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

When I enter a node to edit it (ctrl + double-click) and click outside (it can be single click or double-click)  the node it gets cleared out of its arguments.

https://user-images.githubusercontent.com/108271897/236143038-94391e79-37fa-4cb7-9191-2d7097b54423.mp4

### Expected behaviour

When clicked outside I exit the node edit mode w/o altering it's current state.

### How we can reproduce it?

as described in the "What issue are you facing" field - but it's not a 100% repro rate. It happens rarely, I could reproduce it several times but with many attempts just after spotting this bug coming back.

### Enso Version

2023.5.3 nightly

### Browser or standalone distribution

Standalone distribution

### Browser Version or standalone distribution

standalone

### Operating System

Windows

### Operating System Version

Win11pro 22H2 22621.1555

### Hardware you are using

12th Gen Intel(R) Core(TM) i9-12900HK / RTX3060 Laptop / Nvidia Drivers 531.68

**Comment by 4e6** [comment]
Removing the `--regression` tag because it's very hard to find the point when this behavior started to appear

**Comment by vitvakatu** [comment]
Possibly a duplicate of #6411. Let's check it after #6594 lands

**Comment by vitvakatu** [comment]
Can not reproduce, fixed by #6594
